extensions/README.md

```diff
@@ -103,40 +147,66 @@ If you need to modify the build process, bear in mind that config files are also
-- `extensions/blocks/jukebox/jukebox.php`
-- `extensions/blocks/jukebox/index.js`
-- `extensions/blocks/jukebox/editor.js`
-- `extensions/blocks/jukebox/editor.scss`
-- `extensions/blocks/jukebox/edit.js`
+### Jetpack plugin sidebar
 
-Additionally, the slug of the new block will be added to the `beta` array in the file `extensions/index.json`.
-Since it's added to the beta array, you need to load the beta blocks as explained above to be able to test this block.
+Jetpack adds its own [plugin sidebar](https://wordpress.org/gutenberg/handbook/designers-developers/developers/tutorials/plugin-sidebar-0/plugin-sidebar-1-up-and-running/) to the block editor. You can find it by choosing "Jetpack" from block editor's ellipsis menu or by pressing Jetpack icon in the "pinned plugins" toolbar.
 
-### Examples
+The sidebar itself is always registered in the editor and filled using [Slot Fill](https://github.com/WordPress/gutenberg/tree/master/packages/components/src/slot-fill#readme) mechanism.
 
-`wp jetpack scaffold block "Cool Block"`
+Just render with `JetpackPluginSidebar` component from anywhere in your plugin's code:
 
-`wp jetpack scaffold block "Amazing Rock" --slug="good-music" --description="Rock the best music on your site"`
+```jsx
+import JetpackPluginSidebar from '../../shared/jetpack-plugin-sidebar';
 
-`wp jetpack scaffold block "Jukebox" --keywords="music, audio, media"`
+<JetpackPluginSidebar>
+	<PanelBody title={ __( 'My sidebar section', 'jetpack' ) }>
+		<p>Jetpack is Bestpack!</p>
+	</PanelBody>
+</JetpackPluginSidebar>
+```
+
+The sidebar won't show up at all if nothing isn't rendering in the sidebar's "slot".
+
+Remember to be mindful of posts, pages, custom post types, and re-usable block post type.
+
+See Publicize and Shortlinks for examples how to limit functionality only to some specific post types or posts. Likes & Shares extensions are a great example of how to output content from several extensions to one sidebar section using "slots".
+
+### i18n
+
+As of 04/2019, `wp.i18n` [doesn't support React elements in strings](https://github.com/WordPress/gutenberg/issues/9846). You will have to structure your copy so that links and other HTML can be translated separately.
+
+Not possible:
+
+> Still confused? Check out <a>documentation</a> for more!
+
+Possible:
+
+> Still confused? <a>Check out documentation for more!</a>
+
+### Colors
+
+- To stay consistent with Gutenberg, your extensions should follow [Gutenberg styles and visuals]. Use Gutenberg color variables where possible.
+- The build also supports [Muriel colors](https://github.com/Automattic/color-studio) via SASS variables (`$muriel-pink-300`) and CSS custom properties (`var( --muriel-pink-300 )`). Prefer CSS custom properties if possible.
+
+### Icons
 
+Please use outline versions of [Material icons](https://material.io/tools/icons/?style=outline) to stay in line with Muriel guidelines. Don't rely on icons used in the core to avoid visual mixing up with core blocks.
```

Comment:
```suggestion
Please use outline versions of [Material icons](https://material.io/tools/icons/?style=outline) to stay in line with Muriel guidelines. Don't rely on icons used in core to avoid visual mixing up with core blocks.
```

I think we typically refer to [WordPress] Core without an article :grimacing: